exe_10.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+
+def sort_axis(array:np, axis1:int)->np:
+    """
+    :param array: Some array
+    :param axis1: Sets the sort.
+    :return: array who sorted along the first, last axis of an array.
+    """
+    print(np.sort(array, axis=axis1))

```

Comment:
Seperate IO from logic -
 don’t use  print in functions unless there’s a good reason